GCC-4 fix:

*.c:*:     error: static declaration of 'gl_default_cdio_log_handler' follows non-static declaration
util.h:99: error: previous declaration of 'gl_default_cdio_log_handler' was here

 cd-drive.c |    2 --
 cd-info.c  |    1 -
 iso-info.c |    2 --
 3 files changed, 5 deletions(-)

diff -uNr libcdio-0.72/src.orig/cd-drive.c libcdio-0.72/src/cd-drive.c
--- libcdio-0.72/src.orig/cd-drive.c	2005-01-22 23:21:36.000000000 +0100
+++ libcdio-0.72/src/cd-drive.c	2005-02-06 18:09:16.897350256 +0100
@@ -131,8 +131,6 @@
 
 /* CDIO logging routines */
 
-static cdio_log_handler_t gl_default_cdio_log_handler = NULL;
-
 static void 
 _log_handler (cdio_log_level_t level, const char message[])
 {
diff -uNr libcdio-0.72/src.orig/cd-info.c libcdio-0.72/src/cd-info.c
--- libcdio-0.72/src.orig/cd-info.c	2005-01-27 04:10:06.000000000 +0100
+++ libcdio-0.72/src/cd-info.c	2005-02-06 18:09:58.466030856 +0100
@@ -369,7 +369,6 @@
 
 /* CDIO logging routines */
 
-static cdio_log_handler_t gl_default_cdio_log_handler = NULL;
 #ifdef HAVE_CDDB
 static cddb_log_handler_t gl_default_cddb_log_handler = NULL;
 #endif
diff -uNr libcdio-0.72/src.orig/iso-info.c libcdio-0.72/src/iso-info.c
--- libcdio-0.72/src.orig/iso-info.c	2005-01-22 23:21:36.000000000 +0100
+++ libcdio-0.72/src/iso-info.c	2005-02-06 18:10:09.677326480 +0100
@@ -145,8 +145,6 @@
 
 /* CDIO logging routines */
 
-static cdio_log_handler_t gl_default_cdio_log_handler = NULL;
-
 static void 
 _log_handler (cdio_log_level_t level, const char message[])
 {
